toolchain/install/BUILD

```
@@ -230,3 +231,10 @@ pkg_tar_and_test(
     ],
     test_install_marker = ":install_marker",
 )
+
+# Support `bazel run` on specific binaries.
+run_tool(
+    name = "carbon.bin",
```

I expected `carbon.bin` to name a binary target (as in, a file, or some data).

WDYT about:
```suggestion
    name = "run_carbon",
```

---

Done